csharp/src/SeedLang.Shell/Program.cs

```diff
@@ -32,7 +32,7 @@ internal class Options {
 
       [Option('v', "visualizers", Required = false,
               Default = new VisualizerType[] { VisualizerType.Eval },
-              HelpText = "The Visualizers to be enabled.")]
+              HelpText = "The Visualizers to be enabled. Use All to enable all visualizers.")]
```

Comment:
As for this option, it's really hard for users to figure out how to turn on more than one visualizers via commandline.

E.g., it seems the following commandlines work if we'd like to turn on Binary and Eval:

```
dotnet run --project src/SeedLang.Shell -- -v Binary Eval 
dotnet run --project src/SeedLang.Shell -- --visualizers=Binary Eval
```

But the followings do not work:

```
dotnet run --project src/SeedLang.Shell -- --visualizers="Binary Eval"
dotnet run --project src/SeedLang.Shell -- --visualizers="Binary,Eval"
dotnet run --project src/SeedLang.Shell -- --visualizers=Binary,Eval 
```

Can we have a better HelpText to describe the correct usage of providing more than one visualizers?

Reply:
Add help text description manually, because the CommandLineParser couldn't generate help text for sequence type options.
Also change the separator to "," which might be better than the space.